tests/opentelemetry-docker-tests/tests/redis/test_redis_functional.py

```diff
@@ -37,7 +37,6 @@ def tearDown(self):
     def _check_span(self, span, name):
         self.assertEqual(span.name, name)
         self.assertIs(span.status.status_code, trace.StatusCode.UNSET)
-        self.assertEqual(span.attributes.get(SpanAttributes.DB_NAME), 0)
```

Comment:
add check for attribute RD_REDIS_DATABASE_INDEX

Reply:
Ok, done.